nurseconnect/views.py

```diff
@@ -108,19 +154,44 @@ def form_valid(self, form):
                 question=question,
                 answer=answer
             )
+        self.request.session["registration-step"] = 3
+        return HttpResponseRedirect(reverse("user_register_clinic_code"))
+
+    def get_form_kwargs(self):
+        kwargs = super(
+            RegistrationSecurityQuestionsView, self
+        ).get_form_kwargs()
+        kwargs["questions"] = models.SecurityQuestion.objects.all()
+        return kwargs
+
+
+class RegistrationClinicCodeView(FormView):
+    form_class = forms.RegistrationClinicCodeForm
+    template_name = "registration/register_clinic_code.html"
+
+    def form_valid(self, form):
+        clinic_code = form.cleaned_data["clinic_code"]
+
+        username = self.request.session["username"]
+        user = User.objects.filter(username__iexact=username).first()
 
         # Save clinic code
         user.profile.for_nurseconnect.clinic_code = clinic_code
         user.profile.for_nurseconnect.save()
+        self.request.session["registration-step"] = 0
```

Comment:
Its initialize with a null or false parameter. Since its completed the registration process.